OpenRA.Mods.Common/Traits/Crushable.cs

```diff
@@ -49,7 +49,7 @@ public void WarnCrush(Actor crusher)
 		public void OnCrush(Actor crusher)
 		{
 			Sound.Play(info.CrushSound, crusher.CenterPosition);
-			var wda = self.TraitOrDefault<WithDeathAnimation>();
+			var wda = self.TraitsImplementing<WithDeathAnimation>().FirstOrDefault(s => s.Info.CrushedSequence != null);
```

Comment:
I've noticed that you like linebreaks, but that's not long enough to need one. :/